### ISSUE TITLE
Nouveau modèle de calcul

### ISSUE THREAD

**Issue by mavax74** [mavax74/Projet-2-Informatique-aleatoire#6]
Du coup j'ai refait un nouveau modele, qui prend en compte les occurences des mots dans les docs, et ça permet d'avoir des perfs à 75% mais avec un modèle pas naif, et j'ai aussi corrigé une formule du prof, depuis un moment ça me trottait dans la tête qu'elle était pas super correcte et en discutant avec karim cet aprem j'y ai repensé et je l'ai corrigé, le modèle marche mieux et il est plus clean du coup je vais en reparler sur le rapport

**Comment by mavax74** [comment]
Et \newline, je sais parfaitement comment ça marche, tu les mets n'importe ou, n'importe comment, et ça revient à la ligne, non ?

**Comment by Schwingg** [comment]
ok pour la formule et ton nouveau modele de calcul

\newline et \\ font plus ou moins la meme chose mais ca reviens a forcer la main a latex et du coup il ne fait pas la mise en page du texte en justifie. si tu veux bien faire dans la plus par des cas il suffit de faire:

blabla paragraphe 1 sedgsdgdfg

blabla paragraphe 2 sdgdfgdgf

**Comment by Schwingg** [comment]
je vaix manger je reviens dans 30-45 minutes

**Comment by mavax74** [comment]
ok, merci pour le truc , ca me permettra de moins me faire chier en latex à l'avenir ... :)

**Comment by Schwingg** [comment]
tu as vu mes modifs??

**Comment by mavax74** [comment]
yes, niquel, je vais faire un petit passage orthographe, et je rajouterai une section "bonus" ou j'expliquerai vite fait les modifs faites pour arriver au nouveau modèle de calcul.

T'as vu mon makefile ??? mouahahahah

**Comment by mavax74** [comment]
n'empeche que rien que de lire tes makefile, ben ça me permet de mieux comprendre comment ça marche :)

**Comment by mavax74** [comment]
du coup je l'envoie ?

**Comment by mavax74** [comment]
envoyé, j'ai recompilé l'archive et exécuté, tout à l'air de bien fonctionner

**Comment by Schwingg** [comment]
quel makefile??

cool et en plus tu a pensé a verfié que tout fonctionne dans l'archive avant de l'envoyer
